src/content/tutorial/react-step-5/react-step-5.mdx

```diff
@@ -3,68 +3,216 @@ title: 5. Deploying to IBM Cloud
 - [Create manifest file](#create-manifest-file)
 - [Create static file](#create-static-file)
-- [Deploy](#deploy)
+- [Deploy app](#deploy-app)
+- [Submit pull request](#submit-pull-request)
 
 </AnchorLinks>
 
-### IBM Cloud
+## Preview
 
-[Create an IBM Cloud account](https://cloud.ibm.com/registration) if you don't already have one.
+A [preview](https://react-step-6--carbon-tutorial.netlify.com) of what you'll build (visually no different, but built for production):
 
-### Build for production
+<Row>
+  <Column offsetLg="4">
+    <iframe
+      height="400"
+      title="Carbon Tutorial Step 5"
+      src="https://react-step-6--carbon-tutorial.netlify.com"
+      frameborder="no"
+      allowtransparency="true"
+      allowfullscreen="true"
+      class="bx--iframe bx--iframe--border"
+    />
+  </Column>
+</Row>
 
-Before we deploy our app, we need create an optimized production build
+## Fork, clone and branch
+
+This tutorial has an accompanying GitHub repository called [carbon-tutorial](https://github.com/carbon-design-system/carbon-tutorial) that we'll use as a starting point for each step. If you haven't forked and cloned that repository yet, and haven't added the upstream remote, go ahead and do so by following the [step 1 instructions](/tutorial/react-step-1#fork-clone--branch).
+
+### Branch
+
+With your repository all set up, let's check out the branch for this tutorial step's starting point.
+
+```bash
+$ git fetch upstream
+$ git checkout -b react-step-5 upstream/react-step-5
+```
+
+_Note: This builds on top of step 4, but be sure to check out the upstream step 5 branch because it includes the static assets required to get through this step._
+
+### Build and start app
+
+Install the app's dependencies (in case you're starting fresh in your current directory and not continuing from the previous step):
+
+```bash
+$ yarn
+```
+
+Then, start the app:
+
+```bash
+$ yarn start
+```
+
+You should see something similar to where the [previous step](/tutorial/react-step-4) left off.
+
+## Create IBM Cloud account
+
+Before we get started, [create an IBM Cloud account](https://cloud.ibm.com/registration) if you don't already have one, as we'll be deploying there in a bit.
+
+## Optimize Sass
+
+So far we've be developing in a, well, development environment where static asset optimization hasn't been a priority. If you reference `/src/index.scss`, you'll see one `@import` that is pulling in Carbon's full Sass build.
+
+##### src/index.scss
+
+```scss
+$feature-flags: (
+  ui-shell: true,
+  grid-columns-16: true,
+);
+
+@import 'carbon-components/scss/globals/scss/styles.scss';
+```
+
+To give you an idea of what's all included, open up `node_modules/carbon-components/scss/globals/scss/styles.scss`. You'll see imports for components like accordion, slider, tooltip, etc. Since we aren't using those components, let's exclude them from our built stylesheets. Keeping the `$feature-flags` Sass map, replace the `styles.scss` import with:
```

Comment:
Not super important, but might be interesting to some. It looks like we save about `100kb` by changing the import from `styles.scss` to the scoped imports.

Reply:
Ya, I'm thinking we can go over filesize savings in intro video.

Reply:
Perfect 👍